src/hooks/useSearchResults.ts

```diff
@@ -42,6 +42,7 @@ export const useSearchResults = ({ searchQuery, limit = 50, videoWhereInput }: S
       },
     },
     {
+      fetchPolicy: 'no-cache',
```

Comment:
Why `no-cache`? This will force us to make network request on every filter change even if we already know the result, right?